ignite/metrics/precision.py

```diff
@@ -93,153 +179,139 @@ class Precision(_BasePrecisionRecall):
         .. include:: defaults.rst
             :start-after: :orphan:
 
-        Binary case
+        Binary case. In binary and multilabel cases, the elements of
+        `y` and `y_pred` should have 0 or 1 values.
 
         .. testcode:: 1
 
-            metric = Precision(average=False)
+            metric = Precision()
+            weighted_metric = Precision(average='weighted')
             metric.attach(default_evaluator, "precision")
-            y_true = torch.Tensor([1, 0, 1, 1, 0, 1])
+            weighted_metric.attach(default_evaluator, "weighted precision")
+            y_true = torch.Tensor([1, 0, 1, 1, 0, 1]).long()
```

Comment:
@sadra-barikbin please can you check and replace everywhere in the docstrings (Precision, Recall) that we create tensors like `torch.tensor` and not `torch.Tensor`. Thanks !